content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,459 @@
-#. Set the user as Legal Party for the VAT number you wish to configure the electronic adress.
+#. Go to https://ivaservizi.agenziaentrate.gov.it/portale/ and authenticate;
+#. Go to section :menuselection:`Fatture e Corrispettivi`;
+#. Set the user as Legal Party for the VAT number you wish to configure the electronic address;
 #. In :menuselection:`Servizi Disponibili --> Fatturazione Elettronica --> Registrazione
-   dell’indirizzo telematico dove ricevere tutte le fatture elettroniche`, input Odoo's Codice
-   Destinatario (**K95IV18**), then confirm.
+   dell’indirizzo telematico dove ricevere tutte le fatture elettroniche`, insert Odoo's
+   :guilabel:`Codice Destinatario` `K95IV18`, and confirm.
 
-Give Odoo permission to process files
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Electronic Data Interchange (EDI)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Since the files are transmitted through Odoo's server before being sent to SDICoop or received by
-your database, you need to authorize Odoo to process your files from your database.
+Odoo uses the **FatturaPA** :abbr:`EDI (Electronic Data Interchange)` format for the Italian
+localization, and is enabled by default when installed. When the **file processing authorization**
+has been set, all **invoices** and **bills** are automatically sent for journals with
+:guilabel:`Electronic invoicing` enabled (:menuselection:`Accounting --> Configuration --> Journals
+--> (Select a journal) --> Advanced Settings --> Electronic Data Interchange)`.
 
-To do this, go to :menuselection:`Accounting --> Settings --> Electronic Invoicing` and click on
-*Register*.
+You can check the current status of an invoice by the :guilabel:`Electronic invoicing` field *(cf.
+screenshot below)*. The XML file can be found in the **chatter** of the invoice.
 
-.. image:: italy/register.png
+.. image:: italy/italy-test.png
    :align: center
-   :alt: Register button to allow Odoo to process invoices.
+   :alt: Electronic invoicing status (waiting for confirmation)
+
+.. seealso::
+   :doc:`../../receivables/customer_invoices/electronic_invoicing`
+
+File processing authorization (Odoo)
+------------------------------------
+
+Since the files are transmitted through Odoo's server before being sent to the :abbr:`SdI (Sistema
+di Interscambio)` or received by your database, you need to authorize Odoo to process your files
+from your database. To do so, go to :menuselection:`Accounting --> Configuration --> Settings -->
+Electronic Document Invoicing`.
 
-.. Note::
-   All your files are encrypted upon reception in such a way that only you are able to decrypt them.
+There are **three** modes available:
 
-Enable FatturaPA on the Sales journal
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- :guilabel:`Demo` mode simulates an environment in which invoices are sent to the government. In
+  this mode, invoices need to be *manually* downloaded as XML files and uploaded to the **Agenzia
+  delle Entrate**'s website;
```

Comment:
```suggestion
- :guilabel:`Demo` mode simulates an environment in which invoices are sent to the government. You
  could use this mode to *manually* download the generated XML files and upload to the **Agenzia
  delle Entrate**'s website, but it is mainly meant for demonstration/testing purposes. 
```
Still, we do not want to give people bad ideas...

Reply:
It's the only way to handle B2G at the moment, so I don't consider it a bad idea.